### PR TITLE
Entity ctors

### PR DESCRIPTION
**Pull request by Uitiam**

### CODE REVIEW

File: src/game/GameManager.cpp [hunk 1]
```diff
@@ -74,19 +75,42 @@ void GameManager::updateZombies(const float delta) {
 // Create marine add it to manager, returns marine id
 int32_t GameManager::createMarine() {
     const int32_t id = generateID();
-    marineManager[id] = Marine();
+    SDL_Rect marineRect;
+    SDL_Rect moveRect;
+    SDL_Rect projRect;
+    SDL_Rect damRect;
+
+    marineRect = createRect(marineRect,0,0,100,100);
```
Comment (isaacmorneau) [inline src/game/GameManager.cpp]: all `createRect` can be replaced by `{0, 0, 100, 100}`

File: src/game/GameManager.cpp [hunk 2]
```diff
@@ -422,3 +507,12 @@ bool GameManager::createZombieWave(SDL_Renderer* gRenderer, const int n){
     return true;
 
 }
+
+SDL_Rect& GameManager::createRect(SDL_Rect &rect, int x, int y, int w, int h){
```
Comment (isaacmorneau) [inline src/game/GameManager.cpp]: can now be removed

File: src/sprites/LTexture.h [hunk 3]
```diff
@@ -38,7 +38,8 @@ class LTexture {
     void setAlpha( Uint8 alpha );
 
     //Renders texture at given point
-    void render(SDL_Renderer* gRenderer, int x, int y, SDL_Rect* clip = NULL, double angle = 0.0, SDL_Point* center = NULL, SDL_RendererFlip flip = SDL_FLIP_NONE ) const;
+    void render(SDL_Renderer* gRenderer, int x, int y, SDL_Rect* clip = NULL, double angle = 0.0,
```
Comment (isaacmorneau) [inline src/sprites/LTexture.h]: should be `nullptr`

**Comment by Uitiam** [comment]
Mike's rendering fixes should fix base issue

**Pull request merged**